ui/app/models/pki/role.js

```diff
@@ -29,13 +29,15 @@ export default class PkiRoleModel extends Model {
 
   @attr('string', {
     label: 'Issuer reference',
+    detailsLabel: 'Issuer',
```

Comment:
This is a new attribute option, which will override the label on the details page only